packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -348,6 +348,7 @@ class DateTimePicker extends React.Component {
     // If both date and time are valid, check if the time is the missing hour and invoke onChange.
     // If the date is valid but time is invalid, the time in the dateTime state needs to be cleared and render.
     if (validDate && validTime) {
+      // this.handleChange(event, time);
```

Comment:
This can be removed.

Reply:
removed : [commented line](https://github.com/cerner/terra-framework/blob/4ee766a28a07578ea5a76a52bf85cdfded0516f7/packages/terra-date-time-picker/src/DateTimePicker.jsx#L351)